include/z64.h

```diff
@@ -222,15 +222,15 @@ typedef struct {
 } TargetContext; // size = 0x98
 
 typedef struct {
-    /* 0x00 */ void*    texture;
-    /* 0x04 */ s16      unk_4;
-    /* 0x06 */ s16      unk_6;
-    /* 0x08 */ u8       unk_8;
-    /* 0x09 */ u8       unk_9;
-    /* 0x0A */ u8       delayA;
-    /* 0x0B */ u8       delayB;
-    /* 0x0C */ s16      unk_C;
-    /* 0x0E */ s16      unk_E;
+    /* 0x00 */ u8*      texture;
```

Comment:
Imo this should remain a `void*`, since we don't really care if it's a `u8*`, `u64*` or something else.

In general I think we should use `void*` for pointers to textures whenever we can, so we avoid casts and limit changes if we end up using a different type for these textures later. In fact, we currently use `u64[]` for ZAPD extracted textures, but there are already discussions about changing them to `u8[]` or making it dependent on the type of texture, so it's pretty likely to change in the future.